engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/util/service/query/ServiceQuery.scala

```diff
@@ -24,14 +24,12 @@ import pl.touk.nussknacker.engine.variables.GlobalVariablesPreparer
 import scala.concurrent.{ExecutionContext, Future}
 
 // TODO: Processes using Flink's RuntimeContex, ex. metrics throws NPE, but in another thread, so service works.
-class ServiceQuery(modelData: ModelData) {
+class ServiceQuery(modelData: ModelData, ctx: Context = Context("")) {
```

Comment:
Do we want it here, or in the invoke method?

Reply:
You're right, it would be better to have it in the `invoke` method, but it scala syntax does not think so..

Unless we ged rid of varargs paramter from `def invoke(serviceName: String, args: (String, Expression)*)` or make `context` implicit arg

Reply:
I assumed thats this parameter should have default value..